src/components/TabNav/TabNav.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { ArgsTable, Canvas, Props, Story } from '@storybook/addon-docs/blocks';
+import { TabNav, TabNavItem } from './TabNav';
+
+<Meta title="TabNav" component={TabNav} />
+
+# TabNav and TabNavItem
+
+Navigational components for linking to sibling pages. Use `TabNav` for the container element and `TabNavItem` for individual navigational items.
+
+A link (`<a>`) or button (`<button>`) element should be passed into each `TabNavItem` as a child node.
+
+## Usage
+
+```js
+import { TabNav, TabNavItem } from '@quartz/interface/src/components';
+```
+
+## Usage guidelines
+
+- **Do** use `TabNav` to present sibling pages.
+- **Do** use `TabNav` inside the `Header` component when possible.
+- **Do not** use the `TabNavItem` component without an immediate `TabNav` parent.
+- **Do not** apply `isActive={true}` to more than one `TabNavItem` at a time.
+- **Do not** use a `TabNav` to link to pages that are not semantically related.
+
+## Props
+
+### TabNav
+
+<Props of={TabNav} />
```

Comment:
what do you think about using ArgsTable so that designers and others can use Controls straight from the docs page?

Reply:
In this instance I don't think it's worth it because the only prop is `children`, which is always going to be a big bunch of JSX